File: csharp/difference-of-squares/DifferenceOfSquares.cs
using System;

public static class DifferenceOfSquares
{
    public static int CalculateSquareOfSum(int max)
    {
        int squareOfSum = 0;
        for (int i = 0; i <= max; i++)
        {
            squareOfSum += i;
        }
        return squareOfSum * squareOfSum;
    }

    public static int CalculateSumOfSquares(int max)
    {
        int sumOfSquares = 0;
        for (int j = 0; j <= max; j++)
        {
            sumOfSquares += j * j; 
        }
        return sumOfSquares;
    }

    public static int CalculateDifferenceOfSquares(int max)
    {
        return CalculateSquareOfSum(max)-CalculateSumOfSquares(max);
    }
}

File: javascript/difference-of-squares/difference-of-squares.js
//
// This is only a SKELETON file for the 'Difference of Squares' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export class Squares {
  constructor(num) {
    this.max = num;
  }

  get sumOfSquares() {
    let _sumOfSquares = 0;
    for(let i = 1; i <= this.max; i++){
      _sumOfSquares += i * i;
    }
    return _sumOfSquares;
  }

  get squareOfSum() {
    let _squareOfSum = 0;
    for(let i =  1; i <= this.max; i++){
      _squareOfSum += i;
    }
    return _squareOfSum *_squareOfSum;
  }

  get difference() {
    let answer = this.squareOfSum - this.sumOfSquares
    return answer;
  }
}
